backend/plugins/frontline_api/src/modules/inbox/graphql/resolvers/mutations/conversations.ts

```
@@ -168,21 +227,101 @@ export const conversationMutations = {
+      userId: user._id,
+    };
+
+    const response = await sendConversationToServices(
+      subdomain,
+      integration,
+      serviceName,
+      payload,
+    );
+
+    // if the service runs separately & returns data, then don't save message inside inbox
+    if (response && response.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)